views/pages/experimental/apply/form.njk

```diff
@@ -0,0 +1,122 @@
+{% extends "../../../layouts/main.njk" %}
+
+{% macro errorList(errors, shouldBeLinked)  %}
+    {% if errors | length > 0 %}
+        <ol class="error-list">
+            {% for error in errors %}
+            <li>
+                {% if shouldBeLinked %}<a href="#form-field-{{ error.param }}">{% endif %}
+                {{ error.msg }}
+                {% if shouldBeLinked %}</a>{% endif %}
+            </li>
+            {% endfor %}
+        </ol>
+    {% endif %}
+{% endmacro %}
+
+{% macro inputText(field) %}
+    <label class="ff-label" for="field-{{ field.name }}">{{ field.label }}</label>
+    <input
+        class="ff-text"
+        type="text"
+        id="field-{{ field.name }}"
+        name="{{ field.name }}"
+        value="{% if field.value %}{{ field.value }}{% endif %}"
+        {# {% if field.isRequired %}required{% endif %} #}
```

Comment:
Could maybe add the `aria-required` attribute here too, but given this isn't intended to be user-facing it's probably not worth it for now.